cypress/integration/features.spec.js

```diff
@@ -0,0 +1,122 @@
+describe("FAS Features", () => {
```

Comment:
Since this is a public test suite that we've shared with [scalar](https://github.com/anvc/scalar) folks, we might want to name this more explicitly (e.g. `Harvard-specific Features`) and also add a way to execute these tests conditionally since these wouldn't apply to a standard/vanilla Scalar installation .

According to the docs, Cypress supports [environment variables](https://docs.cypress.io/guides/guides/environment-variables#Setting) and in particular you can create a `cypress.env.json` that is git-ignored, so perhaps we could use that to determine whether or not to skip these tests (e.g. some sort of harvard settings flag). I noticed that there are examples of [including/excluding tests](https://mochajs.org/#inclusive-tests) on the Mocha docs, but not sure if that will work in Cypress.